docs/en/02_Developer_Guides/10_Email/index.md

```diff
@@ -14,15 +14,43 @@ covers how to create an `Email` instance, customise it with a HTML template, the
 Silverstripe CMS provides an API over the top of the [SwiftMailer](http://swiftmailer.org/) PHP library which comes with an
 extensive list of "transports" for sending mail via different services. 
 
-Out of the box, Silverstripe CMS will use the built-in PHP `mail()` command via the `Swift_MailTransport` class. If you'd
-like to use a more robust transport to send mail you can swap out the transport used by the `Mailer` via config:
+For legacy reasons, Silverstripe CMS 4 defaults to using the built-in PHP `mail()` command via a deprecated class `Swift_MailTransport`. However, using this layer is less secure and is strongly discouraged.
+
+It's highly recommended you upgrade to a more robust transport for additional security. The Sendmail transport is the most common one. The `sendmail` binary is widely available across most Linux/Unix servers.
+You can use any of the Transport classes provided natively by SwiftMailer. There are also countless PHP libraries offering custom Transports to integrate with third party mailing service:
+- read the [SwiftMailer Transport Types documentation](https://swiftmailer.symfony.com/docs/sending.html#transport-types) for a full list of native Transport
+- search [Packagist for SwiftMailer Transport](https://packagist.org/?query=SwiftMailer+Transport) to discover additional third party integrations
+
+To swap out the transport used by the `Mailer`, create a file `app/_config/email.yml`
+
+To use a `sendmail` binary:
 
 ```yml
+---
+Name: myemailconfig
+After:
+  - '#emailconfig'
+---
 SilverStripe\Core\Injector\Injector:
   Swift_Transport: Swift_SendmailTransport
 ```
 
-For example, to use SMTP, create a file `app/_config/email.yml`:
+Which will default to calling `/usr/sbin/sendmail -bs`.  You can change the command used for sendmail by updating constructor argument. For instance to use mailcatcher on dev environments, also add the following:
+```yml
+---
+Name: mydevemailconfig
+After:
+  - '#emailconfig'
+Only:
+  environment: dev
+---
+SilverStripe\Core\Injector\Injector:
+  Swift_Transport: Swift_SendmailTransport
+    constructor:
+      0: '/usr/bin/env catchmail -t'
+```
```

Comment:
I would move this example under a dedicated header after the SMTP example and add a little dissertation about how to set up dev environments. Maybe something like this:

 ### Mailer Configuration for dev environments

You may wish to use a different mailer configuration in your development environment. This can be use to suppress outgoing messages or to capture them for debugging purposes in a service like [MailCatcher](https://mailcatcher.me/).

You can suppress all emails by using the [`Swift_Transport_NullTransport`](https://github.com/swiftmailer/swiftmailer/blob/master/lib/classes/Swift/Transport/NullTransport.php).

```yml
---
Name: mydevemailconfig
After:
  - '#emailconfig'
Only:
  environment: dev
---
SilverStripe\Core\Injector\Injector:
  Swift_Transport:  Swift_Transport_NullTransport
```

If you're using MailCatcher, or a similar tool, you can tell `Swift_SendmailTransport` to use a different binary.

```
```yml
---
Name: mydevemailconfig
After:
  - '#emailconfig'
Only:
  environment: dev
---
SilverStripe\Core\Injector\Injector:
Swift_Transport: Swift_SendmailTransport
    constructor:
      0: '/usr/bin/env catchmail -t'
```

Reply:
Have updated, I've changed the syntax slightly from the shortform

```
  Swift_Transport: Swift_SendmailTransport
    constructor:
```
to 
```
  Swift_Transport:
    class: Swift_SendmailTransport
    constructor:
```
I.e. explicitly saying 'class' - it was not working for me on my local once I specified 'constructor' as well